### DISCUSSION TITLE
The maximum amount of chia coins that will ever exist in the lifetime of the cryptocurrency

### DISCUSSION THREAD

**Discussion by mohsennaseri** [Chia-Network/chia-blockchain#7736] [Q&A] [answered]
if i could explain my question I mean :
How much is The maximum amount of coins that will ever exist in the lifetime of the cryptocurrency. 

21000000(pre-farmed) + (?) = Total Coin.
and whether all this 21000000(pre-farmed) has been farmed for now or not?
at coinmarketcap it shows 21,514,760. Is this right?
but in chiaexplorer it shows: 
Total XCH including the pre-farm :    22,214,946 XCH
Thank you.

**Comment by emlowe** [comment, marked as answer]
https://www.chia.net/faq/#faq-18
TL;DR: There is no limit

After the Chia pre-farm, Chia offers farming rewards of 64 chia every 10 minutes. Over the first 12 years the farming rewards will be halved at the end of each 3rd year. From year 13 to infinity, the rewards will remain constant at 4 chia every 10 minutes leading to ever decreasing inflation rates. Chia’s inflation falls through the 0.50% rate 22 years after mainnet launch.

**Reply by mohsennaseri** [reply]
Thank you @emlowe 
one more question:
whether all this 21000000(pre-farmed) has been farmed for now or not?
